src/Dropdown/DropdownListTrigger.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+import DropdownTrigger from "./DropdownTrigger";
+
+export default class DropdownListTrigger extends DropdownTrigger {
+  handleTrigger(event) {
```

Comment:
Why not use the `handleTrigger` in the superclass?

Reply:
no idea, changed it - good catch!